microsoft/testsuites/dpdk/dpdkvpp.py

```diff
@@ -92,7 +87,7 @@ def _install(self) -> bool:
         elif isinstance(node.os, Fedora) or isinstance(node.os, Suse):
             pkg_type = "rpm"
         else:
-            raise UnsupportedDistroException(self.node.os)
+            raise SkippedException(self.node.os)
```

Comment:
The message won't be clear, if skip like this way. Try `SkippedException(UnsupportedDistroException(self.node.os, "<more clarify description here>"))`

Reply:
The SkippedException doesn't produce meaningful exception message by unsupported os. Please try the example to wrap the unsupported exception.

Reply:
Ohhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh I missed the wrapping

Reply:
will squish commits before merge.